plonetheme/onegovbear/theme/index.html

```diff
@@ -217,6 +218,7 @@ <h1 class="documentFirstHeading">This is a heading 1</h1>
               <!-- closes column-content -->
             </div>
 
+            <h1 class="hiddenStructure">Weitere Informationen</h1>
```

Comment:
Afaik no. We may implement it into the accesibility viewlet? and move it with diazo to the right position??